### ISSUE TITLE
How to run the repo in local environment 

### ISSUE THREAD

**Issue by Nayaker** [Nayaker/AlgoListed#104]
Create a good detailed video of how to run the repo and also create a page on the website in ```client > src > Screens``` by the name **runInLocalEnvironment.js** and show your video!

**Comment by Mohit030802** [comment]
Hi @Nayaker ,

Since a lot of libraries are deprecated, Use ```npm i --force``` rather than using ```npm install```. It will work. If anyone still faces this issue comment on this message.

**Comment by soumyadeeproychowdhury** [comment]
Hi! Could you assign me this would love working on it!

**Comment by Mohit030802** [comment]
Hello @soumyadeeproychowdhury, I truly appreciate your interest in this matter. I'm pleased to inform you that the issue has been resolved. However, if you believe you can make further contributions to it, please feel free to do so. Otherwise, you are welcome to work on different issues within this repository. We are looking forward to your continued contributions.

**Comment by Mohit030802** [comment]
To run the front end of the project follow these steps.

1. Go to the client folder in the terminal  ``` cd client```
2. Now to install the required dependencies use ```npm i --force```.
3. And once the required dependencies are installed just use ```npm start``` to run the repo on your local machine.

**Comment by inbuilt-aura** [comment]
@Nayaker is this issue still open, if yes i would like to solve this issue.

**Comment by Nayaker** [comment]
Yes brother it's open! @inbuilt-aura

**Comment by inbuilt-aura** [comment]
> @Nayaker,then plzz assign me.Would like to work on this.

**Comment by inbuilt-aura** [comment]
@Nayaker, is this done?

**Comment by Nayaker** [comment]
No buddy, you can do it - it's an open issue :)

**Comment by inbuilt-aura** [comment]
@Nayaker where is package.jsoon file??

**Comment by inbuilt-aura** [comment]
@Nayaker how will i run if there is no package.json file??

**Comment by Nayaker** [comment]
There is package.json, share screenshot if you don't see it in yours @inbuilt-aura

**Comment by inbuilt-aura** [comment]
@Nayaker,check again.Here:
![Screenshot_2023-10-19-17-56-39-82_e4424258c8b8649f6e67d283a50a2cbc](https://github.com/Nayaker/AlgoListed/assets/74290697/74b812eb-8a76-4396-af4b-43395a27d325)

**Comment by Nayaker** [comment]
Brother, go inside the client and backend - to get the package.json file separately for frontend and backend.

**Comment by inbuilt-aura** [comment]
@Nayaker, thanks for that

**Comment by inbuilt-aura** [comment]
@Nayaker, can you confirm that are frontend and backend aren't connected?

**Comment by Nayaker** [comment]
You can run them independently

**Comment by inbuilt-aura** [comment]
@Nayaker but in order to check everything, we need to connect both. As sign in or other functionality is working or  not thats why asked.

**Comment by Nayaker** [comment]
Yes correct, just for that page it is required to be connected. But it would work even if you don't connect in the current commit.
